buyer-api/src/blockchain/contractEventSubscribers/batchPaymentsSubscribers.js

```diff
@@ -28,13 +28,23 @@ export const updateBuyerStats = async (
   );
 };
 
+/**
+ * @function storeLockingKeyHashByPayIndex Is triggered when
+ * the payment is registered,
+ * storing locally a relationship between the pay index and
+ * the payment transaction hash
+ * @typedef PaymentRegisteredEventValues
```

Comment:
I think that `typedef` should go outside the jsdocs of the functions, and then use it there, right?

Reply:
vscode at least seem to work if I define the typedef just above using it within the function, check out this SS
![Screen Shot 2019-07-04 at 18 50 03](https://user-images.githubusercontent.com/1702755/60688609-a5748e00-9e8c-11e9-8534-331cf5240f84.png)

Reply:
😲 Interesting, ok!